templates/pools/pools.html

```diff
@@ -40,6 +66,13 @@
           <div id="performance-chart" class="w-100 mb-2 d-none">
             {{ template "performance_chart" .Data }}
           </div>
+          <div class="d-flex justify-content-center">
+            <a role="button" class="btn btn-primary btn-sm d-none d-md-flex" id="performance-chart-btn-hide-all">
```

Comment:
On desktop view the button isn't hidden when switching between the two charts on the page, due to conflicting display properties. I'm curious: Why is this a flexbox? Also, why `a` instead of `button`?